Tests/common_BioSQL.py

```diff
@@ -42,9 +42,11 @@
 def load_biosql_ini(DBTYPE):
     """Load the database settings from INI file."""
     if not os.path.isfile("biosql.ini"):
-        raise MissingExternalDependencyError("BioSQL test configuration"
-                                             " file biosql.ini missing"
-                                             " (see biosql.ini.sample)")
+        raise MissingExternalDependencyError(
+            "BioSQL test configuration"
+            " file biosql.ini missing"
+            " (see biosql.ini.sample)"
+        )
```

Comment:
That string can probably be joined up into two medium lines, or maybe even one line and still fit black's 88 character limit?